policy/scan/scan.proto

```diff
@@ -0,0 +1,110 @@
+syntax = "proto3";
+
+import "motor/inventory/v1/inventory.proto";
+import "motor/providers/provider.proto";
+import "google/protobuf/struct.proto";
+import "policy/cnspec_policy.proto";
+
+package cnspec.policy.scan;
+option go_package = "go.mondoo.com/cnspec/policy/scan";
+
+service Scan {
+  // Run executes a scan immediately and reports the result to Mondoo Platform
+  rpc Run(Job) returns (ScanResult) {}
+  // RunIncognito executes a scan immediately and does not report the results to Mondoo Platform
+  rpc RunIncognito(Job) returns (ScanResult) {}
+  // Schedule put a scan job onto the queue for later execution
+  rpc Schedule(Job) returns (Empty) {}
+
+  // RunAdmissionReview scan a Kubernetes AdmissionReview object
+  rpc RunAdmissionReview(AdmissionReviewJob) returns (ScanResult) {}
+
+  // GarbageCollectAssets wraps the underlying DeleteAssets() to allow
+  // specifying filters to identify and delete assets en masse
+  rpc GarbageCollectAssets(GarbageCollectOptions) returns (Empty) {}
+}
+
+// Defines the possible outputs for a scan. If NONE is used, then no output is returned. If RAW is used,
```

Comment:
I do not see `RAW` as an option. Is something missing here?

Reply:
What should `RAW` be? We have `FULL` for returning the complete object with all the data, `ERRORS` for only returning the errors and `NONE` for returning nothing. That's exactly the same as the current scan API.

Reply:
I don't know what `RAW` should be. I mentioned it, because it is mentioned in the comment.

Reply:
Ahaaa! Good catch... I didn't even read the comment. It is definitely wrong 😄